Models/Flip.cs

```diff
@@ -0,0 +1,25 @@
+
+using System;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+
+namespace Coflnet.Sky.SkyAuctionTracker.Models
+{
+    [DataContract]
+    public class Flip
+    {
+        [IgnoreDataMember]
+        [JsonIgnore]
+        public int Id { get; set; }
+
+        [DataMember(Name = "auctionUUID")]
+        public string AuctionUUID { get; set; }
```

Comment:
please limit the storage need for uuids. if you really want to store them in full as strings please limit them to 32 characters in length and set text encoding.
https://github.com/Coflnet/HypixelSkyblock/blob/separation/Data/Auctions/SaveAuction.cs#L21
better yet just store part of the uuid as long and use that for lookups
https://github.com/Coflnet/HypixelSkyblock/blob/separation/Data/Auctions/SaveAuction.cs#L220
I am generating these uids here:
https://github.com/Coflnet/HypixelSkyblock/blob/791502689bcc277f0e0c244ae2e359c9b485a0eb/Server/Services/AuctionService.cs#L34